installer/roles/create-mobile-client-crd/tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+---
+# tasks file for create-mobile-client-crd
+
+- name: Copy CRD definition to tmp file
+  template:
+    src: ../files/mobile-client-crd.yaml
+    dest: /tmp/mobile-client-crd.yaml  
+
+- name: Create mobileClient CRD
+  shell: "oc create -f /tmp/mobile-client-crd.yaml"
+
+- block:
+  - name: Get admin clusterrole
+    shell: "oc get clusterrole admin -o yaml"
+    register: admin_original_content
+
+  - name: Create new admin clusterrole file
+    template:
+      src: ../files/clusterrole-template.j2
+      dest: /tmp/clusterrole.yaml
+    vars:
+      original_content: "{{ admin_original_content.stdout | from_yaml | to_nice_yaml (indent=2) }}"
+
+  - name: Remove timestamp from admin
+    lineinfile:
+      path: /tmp/clusterrole.yaml
+      regexp: "creationTimestamp:"
+      state: absent
+
+  - name: Replace admin clusterrole
+    shell: "oc replace -f /tmp/clusterrole.yaml"
+
+- block:
+  - name: Get edit clusterrole
+    shell: "oc get clusterrole admin -o yaml"
```

Comment:
shouldn't this be `oc get clusterrole edit -o yaml`?

Reply:
I don't think editing it like in the manual steps in ansible would work. So instead I am getting the current clusterrole, adding the new permissions and then writing it all back to the cluster

Reply:
i mean because in the manual steps there are two roles, admin and edit that need to be changed:

```
oc edit clusterrole admin # add the above and save
oc edit clusterrole edit # add the above and save
```

It looks like you are appending twice to the same file. Or maybe my understanding of the `oc get clusterrole` command is wrong.

Reply:
You are correct, copy and paste error